gateway/setup/create_certs.sh

```diff
@@ -0,0 +1,19 @@
+if ! command -v openssl &> /dev/null
+then
+    echo "openssl could not be found"
+    exit
+fi
+mkdir -p ../certs
+cd ../certs
+# create CA private key
+openssl genrsa -des3 -out ca_key.pem 2048
+# create CA certificate
+openssl req -x509 -new -nodes -key ca_key.pem -sha256 -days 365 -out ca_cert.pem
```

Comment:
This will generate a CA certificate that expires after 1 year - which may give QA some funny issues if they keep using it for 1 year. In general, if a cert or CA is not intended to require renewal, it's better to just put `-days 9999` there.

Also, you can skip the unnecessary subject question prompts by adding `-subj "/CN=Local Hummingbot CA"`.